eks/cluster-loader/clusterloader2/addon.go

```diff
@@ -93,10 +93,19 @@ func (c *ClusterLoader) buildConfigMapData() (map[string]string, error) {
 		bodyString := strings.ReplaceAll(string(body), "\n", "\n    ")
 		dataMap[path.Base(uri)] = bodyString
 	}
+	if len(c.Config.Spec.ClusterLoader.TestOverrides) > 0 {
```

Comment:
You could probably oneline this whole thing like 
```
dataMap["overrides.yaml"] = strings.Join(c.Config.Spec.ClusterLoader.TestOverrides, IndentedNewline)
```

Reply:
gotcha. quick and easy.